Gems/GradientSignal/Code/Source/Editor/EditorGradientBakerComponent.cpp

```diff
@@ -507,6 +520,23 @@ namespace GradientSignal
 
             AZ::TickBus::Handler::BusDisconnect();
 
+            // After a successful bake, if the Entity that contains this gradient baker component also
+            // has an image gradient component, then update the image gradient's image asset with the
+            // output path that we baked to
+            if (AzToolsFramework::EntityHasComponentOfType(GetEntityId(), GradientSignal::EditorImageGradientComponentTypeId))
+            {
+                AzToolsFramework::ScopedUndoBatch undo("Update Image Gradient Asset");
```

Comment:
Can you verify if this is stored as a separate undo node, or if it's part of the change that triggers it?
(Using the ScopedUndoBatch, it should try to merge the change with any other ongoing undo batch... but better double check)

Reply:
Verified this is stored as its own undo node. The change that triggers it doesn't actually get an undo node because it's just writing out to a source asset on disk, it isn't modifying any prefab data.